apps/site/lib/site_web/templates/layout/_desktop_menu_contact_us.html.eex

```diff
@@ -1,4 +1,41 @@
-<div class="menu-row">
+<%= if Laboratory.enabled?(@conn, :deskstop_nav_redesign) do %>
+  <div class="menu-row">
```

Comment:
You can access the variable that contains all the new navigation sub menus and links via `SiteWeb.LayoutView.nav_link_content_redesign(@conn)`. There's an example of iterating through it in a different template [here](https://github.com/mbta/dotcom/pull/1167/files#diff-44b4356f3fd1a67ccc372a538f1e045b3ce4b1a94363f1232436c425970e06b1). So I'd recommend iterating through that to generate this.